Retail.lua

```diff
@@ -370,6 +399,14 @@ addon.Spells = {
     [329543] = { type = BUFF_DEFENSIVE }, -- Divine Ascension
         [328530] = { type = IMMUNITY, parent = 329543 }, -- Divine Ascension
     [335467] = { type = DEBUFF_OFFENSIVE, nounitFrames = true, nonameplates = true }, -- Devouring Plague
+    [453] = { type = CROWD_CONTROL }, -- Mind Soothe
```

Comment:
```suggestion
    [453] = { type = DEBUFF_OFFENSIVE, noraidFrames = true }, -- Mind Soothe
```
Only meaningful in pve, and can't picture a situation where you'd want to see this over anything else.